packages/utils/test/base64.test.ts

```diff
@@ -0,0 +1,44 @@
+import { base64 } from '../src';
+
+describe('base64', () => {
+  it('should successfully encode a ascii character', () => {
```

Comment:
Capture and restore the decoder/encoder?
```suggestion
describe('base64', () => {
  let globalTextEncoder: typeof global.TextEncoder;
  let globalTextDecoder: typeof global.TextDecoder;
  beforeAll(() => {
    globalTextEncoder = global.TextEncoder;
    globalTextDecoder = global.TextDecoder;
  });
  afterAll(() => {
    global.TextEncoder = globalTextEncoder;
    global.TextDecoder = globalTextDecoder;
  });
  it('should successfully encode a ascii character', () => {
```